CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
 
-*And if you need help getting your article converted from it’s current format into Markdown and onto the Quire website, we’re happy to guide you through that process.*
+**Download Your Quire Project from Github**
+1. Open GitHub Desktop.
+2. Click on the upper left-hand corner of the application window, where it says “Current Repository”. Click “Add” and choose the option to “Clone repository…”
+3. Click the URL menu option and cut-and-paste the following URL: https://github.com/thegetty/quire-docs.
+4. Save the cloned `quire-docs` repo in your home directory.
 
-### Improve the Documentation
+**Preview Your Quire Publication**
+1. Next, pull up your [command line](https://quire.getty.edu/learn/tutorial/#1-work-in-a-command-line-shell) to navigate through your computer files.
+2. Use the cd command to navigate to your computer's home directory and then navigate into your `quire-docs` folder.
+3. Once you are in your `quire-docs` folder, run the following commands:
+```
+quire install
+quire preview
+```
+4. Go to [http://localhost:1313](http://localhost:1313) the project as you work on it.
 
-The Quire documentation is hosted in a separate repository along with the website, and can be found [here](https://github.com/thegetty/quire-docs). The Quire documentation is continuously being updated for clarity and completeness, but it’s not always easy to keep up with the pace of Quire’s development!
+**Open Your Project in GitHub Desktop**
+1. At the top right, click "Current Repository" and choose the `quire-docs` repo.
 
-If there’s a section you think it missing or could be improved, we’d love for your help. And the documentation is itself a Quire site, so it should be familiar to work in.
+**Create a New Branch in GitHub Desktop**
+1. Navigate to where it says “Current Branch” at the top of your GitHub Desktop window.
+2. Click the drop-down arrow and select “new branch” to the right of the Filter box.
+3. Name your branch.
+4. Select “main” where it says “Create branch based on.” This means you are creating a new branch that runs parallel to the main project.
 
-[Propose a new section or edit](https://github.com/thegetty/quire-docs/issues)<br />
-[Find an existing issue to work on](https://github.com/thegetty/quire-docs/issues)<br />
-[Read the Quire Documentation Style Guide](https://github.com/thegetty/quire-docs/wiki/Quire-Website-&-Documentation-Style-Guide)
+**Open Project File in Atom**
+1. Open Atom
+2. Click File > Open
+3. Navigate to your home directory and your select `quire-docs` folder.
+5. In Atom, expand the contents folder to see the editable markdown files and look for the file you'd like to make your changes in.
```

Comment:
this is numbered as "5" in the Markdown though it maybe comes out correctly as "4" in the HTML view